java/core/src/java/org/apache/orc/impl/RecordReaderUtils.java

```diff
@@ -103,8 +106,7 @@ public OrcProto.StripeFooter readStripeFooter(StripeInformation stripe) throws I
     public BufferChunkList readFileData(BufferChunkList range,
                                         boolean doForceDirect
                                         ) throws IOException {
-      RecordReaderUtils.readDiskRanges(file, zcr, range, doForceDirect, minSeekSize,
-                                       minSeekSizeTolerance);
+      RecordReaderUtils.readDiskRangesVectored(file, range, doForceDirect);
```

Comment:
Got it. I believe it's okay to use `Hadoop Vectored IO` only from Apache ORC 2.0.0.